scripts/compute_branch.sh

```diff
@@ -1,18 +1,21 @@
 #!/usr/bin/env bash
 
-if [[ -n $(git status --porcelain) ]]; then
-    # If the branch isn't clean, default to HEAD to match old behavior.
-    BRANCH="HEAD"
-elif [ -z "${TRAVIS_BRANCH}" ]; then
-    # if there is no travis branch, set based on tag or branch
-    case "$(git describe --tags)" in
-      *"beta")    BRANCH="rel/beta" ;;
-      *"stable")  BRANCH="rel/stable" ;;
-      *"nightly") BRANCH="rel/nightly" ;;
-      *)          BRANCH=$(git rev-parse --abbrev-ref HEAD)
-    esac
-else
-    BRANCH="${TRAVIS_BRANCH}"
+BRANCH="${BRANCH:-}"
+if [ -n "$BRANCH" ]; then
+    if [[ -n $(git status --porcelain) ]]; then
+        # If the branch isn't clean, default to HEAD to match old behavior.
+        BRANCH="HEAD"
+    elif [ -z "${TRAVIS_BRANCH}" ]; then
+        # if there is no travis branch, set based on tag or branch
+        case "$(git describe --tags)" in
+        *"beta") BRANCH="rel/beta" ;;
+        *"stable") BRANCH="rel/stable" ;;
+        *"nightly") BRANCH="rel/nightly" ;;
+        *) BRANCH=$(git rev-parse --abbrev-ref HEAD) ;;
+        esac
+    else
+        BRANCH="${TRAVIS_BRANCH}"
+    fi
```

Comment:
I thought we weren't using Travis anymore?

Reply:
we are not, but some vars are hard to get rid of.

Reply:
if we change it here, i think we need to change it in whole bunch of other scripts, which might be outside of the scope of this ticket.

Reply:
I suspected that was the case. No worries here.